app/helpers/sessions_helper.rb

```diff
@@ -30,7 +30,7 @@ def remember user
   end
 
   def forget user
-    return unless user.present?
+    return if user.blank?
```

Comment:
dùng như này thì có được ko e?
```suggestion
    return unless user
```

Reply:
Dạ, em đang tìm cách fix cái `rubocop` `auto fixing` của em.